### ISSUE TITLE
[Deep Learning] Gradient Decent & Backpropagation | 6mini.log

### ISSUE THREAD

**Issue by 6mini** [6mini/6mini.github.io#83]
https://6mini.github.io/deep%20learning/2021/10/19/Backpropagation/ 

 경사하강법과 역전파에 대한 이해. Keras Framework 이용 Model 구축
